samples/subsys/mgmt/mcumgr/smp_svr/boards/nrf54l15pdk_ext_flash.overlay

```diff
@@ -0,0 +1,5 @@
+/ {
```

Comment:
Needs full board name in the filename, is this for cpuapp, cpurad, etc.? Also needed for hwmv2 migration which is in progress